src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.cs

```diff
@@ -94,9 +94,19 @@ private static string EscapeIdentifier(string identifier)
 
         public void VisitWithNullability(TypeSymbol symbol, NullableFlowState topLevelNullability)
```

Comment:
>TypeSymbol [](start = 41, length = 10)

Is it still necessary to use `TypeSymbol` in `SymbolDisplay` rather than `ITypeSymbol`, etc.?

Reply:
Currently yes, but after a refactor I have planned no.

---
In reply to: [269217827](https://github.com/dotnet/roslyn/pull/34429#discussion_r269217827) [](ancestors = 269217827)